src/test_l1_dsHdmiIn.c

```diff
@@ -744,7 +751,7 @@ void test_l1_dsHdmiIn_positive_dsHdmiInSelectZoomMode_source(void) {
  * @note Testing for the `dsERR_OPERATION_NOT_SUPPORTED` and `dsERR_OPERATION_FAILED` might be challenging since it requires a specific scenario where the attempted operation is not supported.
  *
  */
-void test_l1_dsHdmiIn_negative_dsHdmiInSelectZoomMode_source(void) {
+void test_l1_dsHdmiIn_negative_dsHdmiInSelectZoomMode(void) {
 
     gTestID = 14;
     UT_LOG("\n In %s [%02d%03d]\n", __FUNCTION__, gTestGroup, gTestID);
```

Comment:
For all the negative cases, if it is a sink device or a source device with no Input ports, the API will return NOT_SUPPORTED and we also need to validate the same in the negative case rather than any other error code.